<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>José Manuel Díez Botella</author>
 <class>imprimediario</class>
 <widget class="QDialog" name="imprimediario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imprimir diario</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="ejerciciogroupBox">
       <property name="title">
        <string>Filtrar por e&amp;jercicio</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="ejercicioscomboBox"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QCheckBox" name="confiltrocheckBox">
         <property name="text">
          <string>Imprimir filtro</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fechaactualcheckBox">
         <property name="text">
          <string>Imprimir fecha actual</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="horizontal_checkBox">
         <property name="text">
          <string>Informe horizontal</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="fechasgroupBox">
       <property name="title">
        <string>Filtrar por fechas</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Fecha inicial:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="inicialdateEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Fecha final:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="finaldateEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="imprime_asiento_checkBox">
     <property name="text">
      <string>Imprime asiento actual</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>51</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="imprimepushButton">
       <property name="text">
        <string>&amp;Informe</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/printer.png</normaloff>:/iconos/graf/printer.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+I</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="visualizarpushButton">
       <property name="text">
        <string>Informe &amp;TEX</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/ghostview.xpm</normaloff>:/iconos/graf/ghostview.xpm</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+V</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="xmlpushButton">
       <property name="text">
        <string>&amp;XML</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/xml.jpeg</normaloff>:/iconos/graf/xml.jpeg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copiarpushButton">
       <property name="text">
        <string>&amp;Copiar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/copiar.png</normaloff>:/iconos/graf/copiar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="csvpushButton">
       <property name="text">
        <string>&amp;Archivo CSV</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/disk.xpm</normaloff>:/iconos/graf/disk.xpm</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="texpushButton">
       <property name="text">
        <string>&amp;Editar TEX</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/tex.png</normaloff>:/iconos/graf/tex.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salirpushButton">
       <property name="text">
        <string>&amp;Salir</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/application-exit.png</normaloff>:/iconos/graf/application-exit.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>ejerciciogroupBox</tabstop>
  <tabstop>ejercicioscomboBox</tabstop>
  <tabstop>confiltrocheckBox</tabstop>
  <tabstop>fechaactualcheckBox</tabstop>
  <tabstop>horizontal_checkBox</tabstop>
  <tabstop>fechasgroupBox</tabstop>
  <tabstop>inicialdateEdit</tabstop>
  <tabstop>finaldateEdit</tabstop>
  <tabstop>imprimepushButton</tabstop>
  <tabstop>visualizarpushButton</tabstop>
  <tabstop>xmlpushButton</tabstop>
  <tabstop>copiarpushButton</tabstop>
  <tabstop>csvpushButton</tabstop>
  <tabstop>texpushButton</tabstop>
  <tabstop>salirpushButton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>salirpushButton</sender>
   <signal>clicked()</signal>
   <receiver>imprimediario</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
